### DISCUSSION TITLE
HPE & Aruba - Standardizing Data

### DISCUSSION THREAD

**Discussion by danner26** [netbox-community/devicetype-library#1137] [Polls]
# **We are making the decision that Aruba devices will be merged into the HPE folder**

Hello all,

It was recently brought to my attention that we have two separate folders, one for HPE and one for Aruba. Since Aruba is a HPE asset, we must decide whether to keep the Aruba gear within its own folder or to consolidate it into the HPE folder. I personally believe the Aruba gear should be moved to the HPE folder for two reasons: 1. Aruba gear must be purchased through the HPE website or an authorized reseller, and 2. Several of the devices display the phrase "Aruba a Hewlett Packard Enterprise Company".

As a result of this decision, other types of devices will also be affected. Dell, for example, offers its own line of networking hardware, which can be found within the Dell folder. The same applies to Cisco Meraki. This scenario is likely to affect other types of devices as well, so please think carefully before making a decision.

Thank you!

**Comment by jsenecal** [comment]
I think it should follow the company documentation, if it's an HPE product it should remain there. If it's now an Aruba product, then move it under Aruba

**Reply by danner26** [reply]
Moreso this poll is about the Aruba folder which is a product line manufactured by HPE

**Comment by jeremystretch** [comment]
I have a _slight_ preference for maintaining a separate Aruba folder. I suspect most users will prefer to model Aruba as a separate manufacturer in their NetBox deployments (though I admit I have no data to back this up).

**Reply by danner26** [reply]
Fair enough, I understand both sides of the conversation hence the poll.. just want to see if there is a consensus among everyone so that we can document and maintain this going forward

Either way there is hardware currently listed in both places so when a decision is made we can migrate the files accordingly 😀

**Reply by jeremystretch** [reply]
Just to reiterate, in the absence of a strong signal from the community, it's 100% your call to make @danner26.

**Comment by fuzzbawl** [comment]
The same argument could be said for Cisco and Meraki. Cisco treats Meraki as a product line at this point just like Catalyst. Does HPE treat Aruba like a product line or more of a child company? If it’s a product line like I feel it is, then makes sense to roll it all into HPE folder.Just my two cents based on what I’ve seen with other folders.

**Reply by danner26** [reply]
Yup that's exactly why I mentioned this affects others as well..

**Comment by francesco-noc** [comment]
I think keeping Aruba (an HPE company) branded gear separated may be a bit misleading because the device details are on both the HPE site and the Aruba site and they can be purchased in HPE store, as stated in first post. This is happening in this repository, some HPE-Aruba devices (same series, e.g. 2530) are in the Aruba folder and others in the HPE folder.

**Reply by danner26** [reply]
And that is what I am trying to solve here, as a community decide where these device definitions should live and stick to that.. If it is Aruba devices in the Aruba folder then great, if it is Aruba devices in the HPE folder then great as well. I just want to fix the split brain issue that we currently have

**Comment by aopdal** [comment]
The current Aruba modules should probably also be moved to HPE, same as the devices?
If you think it's to any help, I may try to make the change and create a PR.

**Reply by danner26** [reply]
Absolutely, I am working on that as well
Just running a bit behind haha

**Reply by danner26** [reply]
#1169 is merged
Thanks for the reminder!
